Update cloud-cli-installation.md (#5423)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/cloud-cli-installation.md

Before you begin, make sure you have [Homebrew installed](http://brew.sh/) in your code editor or command line terminal. Refer to the [FAQs](#faqs) if your operating system runs into path conflicts. 

1. Verify that you don't already have dbt Core installed by running the following command.    
  
  ```bash
  which dbt
  ```
  If the output is `dbt not found`, then that confirms you don't have it installed.

:::tip Run `pip uninstall dbt` to uninstall dbt Core

> If you've installed dbt Core globally in some other way, first uninstall it with:

Author: ```suggestion
If you've installed dbt Core globally in some other way, uninstall it first before proceeding:
```

> If you have installed dbt Core globally in some other way, you'll need to uninstall it first before proceeding. To uninstall dbt Core and remove it from the system, run:

Author: ```suggestion
If you've installed dbt Core globally in some other way, first uninstall it with:
```

```bash

> `pip uninstall dbt`

Author: ```suggestion
pip uninstall dbt
```
